compose/snippets/src/main/java/com/example/compose/snippets/adaptivelayouts/SampleListDetailPaneScaffold.kt

```diff
@@ -49,19 +47,13 @@ import androidx.compose.ui.unit.sp
 @Composable
 fun SampleListDetailPaneScaffoldParts() {
     // [START android_compose_adaptivelayouts_sample_list_detail_pane_scaffold_part02]
-    val navigator = rememberListDetailPaneScaffoldNavigator<Nothing>()
+    val navigator = rememberListDetailPaneScaffoldNavigator<MyItem>()
```

Comment:
Will this correctly save `MyItem` in saved instance state? We aren't using `MyItem.Saver` anywhere now, which seems suspicious.

Reply:
You're right, we aren't actually passing the saver into the rememberX so I guess it has to be a Parcelable class to be safely used here.